common/src/main/java/io/pravega/common/concurrent/ExecutorServiceFactory.java

```diff
@@ -71,20 +68,18 @@
 
         // In all of the below, the ThreadFactory is created in this class, and its toString() returns the pool name.
         if (this.detectionLevel == ThreadLeakDetectionLevel.None) {
-            this.createScheduledExecutor = (size, factory) -> new ScheduledThreadPoolExecutor(size, factory, new CallerRuns(factory.toString()));
+            this.createScheduledExecutor = (size, factory) -> new ThreadPoolScheduledExecutorService(size, size, 100, factory, new AbortPolicy());
```

Comment:
nit: can we make this `100` constant?
should this be passed as in input param?

Reply:
Removed